source/elements/oneDPL/source/parallel_api/buffer_wrappers.rst

```diff
@@ -43,33 +43,56 @@ Buffer Wrappers
 
 ``oneapi::dpl::begin`` and ``oneapi::dpl::end`` are helper functions
 for passing `SYCL`_ buffers to oneDPL algorithms.
-These functions accept a buffer and return an object
-of an unspecified type that satisfies the following requirements:
-
-- it is ``CopyConstructible``, ``CopyAssignable``, and comparable
-  with operators ``==`` and ``!=``;
-- the following expressions are valid: ``a + n``, ``a - n``,
-  ``a - b``, where ``a`` and ``b`` are objects of the type,
-  and ``n`` is an integer value;
-- it provides the ``get_buffer()`` method that returns the buffer
-  passed to the ``begin`` or ``end`` function.
+These functions accept a SYCL buffer and return a *buffer position object*,
+which type satisfies the following requirements but is otherwise unspecified:
+
+- It is copy-constructible and copy-assignable.
+- It is comparable with ``operator==`` and ``operator!=``.
+- It provides the ``get_buffer()`` method that returns the SYCL buffer, which an object was built over.
```

Comment:
```suggestion
- It provides the ``get_buffer()`` method that returns the SYCL buffer, which the object was built over.
```

Reply:
I see your point, we are talking about the type requirements, not requirements on objects of that type.

Do we need to clarify this a little more to say 
"Objects of this type provide the member function ``get_buffer()`` that returns the SYCL buffer over which the object was built."  ?

If so, perhaps the line above it should be
"Objects of this type are comparable with ``operator==`` and ``operator!``."


I guess what I take issue with in the current formulation is that there is not a notion that `get_buffer()` connects directly to the calling object.  It can be inferred, but I don't think it is stated with this language.

Reply:
Should be better now :)